articles/mvc-mvp-mvvm/index.md

```diff
@@ -0,0 +1,52 @@
+### What are Architectural Patterns in Android?
+
+### Introduction
+
+We use various mobile and web applications in our day-to-day life. These applications are aimed to make our computational experience simple and seamless. Some of these applications are compatible with mobile operating systems such as Android, iOS and some are compatible with desktop operating systems such as Windows, macOS, Linux, etc and some of these even may be cross-platform (applications that can be operated on more than one operating system).
+
+Even though these applications may differ in aspects such as what they do, how they work, how it is built, etc.., one thing that remains the same -- that is their **unique architecture and User Interface**. These elements can serve as a key component to the overall application creation process. Having a better and unique UI can make a huge difference in how great an application is. The UI is solely not based on the developers' imagination, but also on the consumer who may be using it. The interface must be user-friendly so that it will be easier for the user to grasp the visual changes taking place in the application while using it.

```

Comment:
> one thing that remains the same

Shouldn't this be two factors remain the same?
Since it's the architecture and the User Interface.